test-project/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.7
+sbt.version=0.13.16
```

Comment:
Why is this still 0.13?

Reply:
Fair point - apologies for missing that.  I'll bump this.